### ISSUE TITLE
Cannot resolve external dependency com.android.tools.build:gradle

### ISSUE THREAD

**Issue by jandebaker** [google/ExoPlayer#6477]
Hello.
When I try to build project from local copy downloaded from git I get following error
ERROR: Cannot resolve external dependency com.android.tools.build:gradle because no repositories are defined.
Required by:
    project :demo

I have already set in Android Studio - Settings - Gradle path to latest gradle version 

![image](https://user-images.githubusercontent.com/50493358/65726771-23c08500-e0b6-11e9-9a78-44c1514ffaa0.png)

Could anyone help?

**Comment by icbaker** [comment]
Are you following the ExoPlayer Hello World tutorial?

Did you add the repositories to your build.gradle as specified here?
https://exoplayer.dev/hello-world.html#add-repositories

**Comment by jandebaker** [comment]
Hi. Yes, I have the repositories to my build.gradle.
But I do not do Hello World Tutorial. I wish to make Exoplayer stand alone app, as it is described here https://exoplayer.dev/demo-application.html
I did local copy of project and trying compile and run app on connected device.

This is screen on project and errors 

![image](https://user-images.githubusercontent.com/50493358/65793006-39d94e80-e165-11e9-8c1c-c6e75b08ec22.png)

**Comment by icbaker** [comment]
I tried to repro from the instructions you linked, and mine builds fine. I did:

1. `$ git clone https://github.com/google/ExoPlayer.git`
2. Open Android Studio
3. File -> New -> Import Project...
4. Select the 'ExoPlayer' directory that was created in step 1 (should have an Android Studio logo) then press 'OK'

One difference I note between your screenshot and the release-v2 branch on github is the version numbers of the gradle & gms strict-version-matcher dependencies. On github these are `3.4.0` and `1.1.0` respectively, so it seems you've introduced some local changes. 

If you try and build immediately after cloning, with no local changes, does it work?
